tools/src/tester/TestRunner.ts

```diff
@@ -69,6 +69,9 @@ export default class TestRunner {
     const path = file === '' ? folder : `${folder}/${file}`
     const next_prefix = prefix === '' ? file : `${prefix}/${file}`
     if (fs.statSync(path).isFile()) {
+      if (!path.endsWith('.yaml')) {
+        return []
+      }
```

Comment:
This is fine, but if you look below there's `EXCLUDED_FILES.includes()`, feels like this is where we should be checking the .yaml extension as well to keep things in one place?

Reply:
Happy to move it, when I looked at it that seemed to be when it was reading files in a directory, so this will catch even the case when a file was used on the command line.

So this would not run:
npm run test:spec -- --opensearch-insecure --tests-path tests/cat/badfile

but that is an edge case I guess. Let me know which you prefer.

Reply:
or maybe i should move the EXCLUDE? Or actually the current list is a file that won't run anymore anyway, lol.

Reply:
Right. So we definitely don't want both checks, pick your favorite implementation and update?